ui/src/TextUpload.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
 import axios from 'axios';
 
-class TextUpload extends React.Component<any, any> {
+type TextProps = {}

```

Comment:
What does setting the TextProps do?

Reply:
This is just creating a data type for the props, for TypeScript's use.  At the moment, this component doesn't receive any props, so there's not much utility for it, but it can be there for future use.